packages/core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -34,8 +35,6 @@ const DEFAULT_LABELS = {
   invalidLabel: "Invalid date",
 };
 
-export type HvDatePickerStatus = "standBy" | "valid" | "invalid";
```

Comment:
Not only this type is being exported to the users, it's being used in our samples. Also, `HvDatePickerProps["status"]` isn't a great alternative since you'll get the `undefined`.
Therefore, this is a type that users are probably using it in the apps. We should deprecate it, instead of removing it

We can do something like:
```tsx
/** @deprecated use `HvFormStatus` instead */
export type HvDatePickerStatus = HvFormStatus;
```

Reply:
Yeah, you're right. I'll update, thanks.